app/src/display/Kconfig

```diff
@@ -6,6 +6,7 @@ menuconfig ZMK_DISPLAY
     select LVGL
 	select LVGL_THEMES
 	select LVGL_THEME_MONO
+    select LVGL_BUILD_IN_FONT_ROBOTO_12
```

Comment:
The widget stuff should be doing this for us in the widgets that need this font. This should be added to the new Kconfig config you create for the new widget.